src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.scss

```diff
@@ -1,110 +1,127 @@
 @import '../../../../styles/variables';
 
 .sendbird-parent-message-info-item {
-  position: relative;
   margin-top: 8px;
+  position: relative;
 }
 
 .sendbird-parent-message-info-item__text-message,
 .sendbird-parent-message-info-item__og-field {
-  padding-right: 4px;
   display: inline-block;
+  padding-right: 4px;
   white-space: pre-line;
   word-break: break-all;
 }
+
 .sendbird-parent-message-info-item__og-field {
   display: inline-flex;
   flex-direction: column;
   margin-top: 4px;
 }
+
 .sendbird-parent-message-info-item__og-field__content {
   display: inline-flex;
   flex-direction: column;
-  padding: 8px 12px;
   gap: 4px;
+  padding: 8px 12px;
 }
 
 .sendbird-parent-message-info-item__file-message {
+  align-items: center;
   display: inline-flex;
   flex-direction: row;
-  align-items: center;
   gap: 8px;
 }
+
 .sendbird-parent-message-info-item__file-message__file-name {
   display: inline-block;
+  max-width: 200px;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
-  max-width: 200px;
 }
 
 .sendbird-parent-message-info-item__thumbnail-message {
-  position: relative;
   display: block;
-  width: 200px;
   height: 148px;
+  position: relative;
+  width: 200px;
 }
+
+.sendbird-parent-message-info-item__multiple-files-message-wrapper {
+  background-color: transparent !important;
```

Comment:
We must refrain from using `!important` as much as possible.
Because this can avoid the customer CSS settings

Reply:
oh thats true.. let me think another solution

Reply:
```
.sendbird-parent-message-info-item__multiple-files-message-wrapper {
  padding: 0;

  @include themed() {
    background-color: transparent;
  }
}
```
I have no clue how above solution works but it works

Reply:
maybe a more specific selector made it work